rest-api/src/main/java/gov/cms/qpp/conversion/api/logging/AttachmentHashPartConverter.java

```diff
@@ -29,8 +26,8 @@ public String convert(ILoggingEvent event) {
 
 		try {
 			hashPart = getHashPart();
-		} catch (IOException | ServletException e) {
-			DEV_LOG.trace("No part to associate with log output.", e);
+		} catch (IOException | ServletException | NullPointerException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Avoid catching NullPointerException. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1696)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)

Reply:
Shush @FlexionSonarQube.

Reply:
@FlexionSonarQube has a point here.  NPE is a runtime exception that we generally don't want to catch.  The other two are checked exceptions.

Reply:
Hmmm, we could just check for `null` around https://github.com/CMSgov/qpp-conversion-tool/pull/598/files#diff-5eed8e91cf01031595a455f39992335fR60 instead.  One of those chained method calls returns `null` which subsequently throws a `NullPointerException` in the early stages of the application's start-up.